frontend/src/tabs/authorship.pug

```diff
@@ -70,8 +70,8 @@
           title="click to view the blame view of file"
           )
             i.button.fas.fa-user-edit
-        pre.hljs
+        pre.hljs(v-if="doLoadFile(file)" v-show="file.active")
```

Comment:
Why v-show and v-if together? Also, the attribute `wasCodeLoaded` seems not useful.

Reply:
Hi!
`v-if` is to prevent vue from unloading it from the dom ( caching it essentially ) once it has been loaded for the first time.
In here, `doLoadFile` returns `wasCodeLoaded || file.active`, and `wasCodeLoaded` is set to true and remains permenantly so once the file is set to `active` for the first time, and has been loaded 
( either by `displayMoreFiles` or the user re-expanding a by-default collapsed file )

`v-show` is just to toggle `display: none` on/off on it

Reply:
I understand. But isn't this what v-show does? It make the elements remain in the DOM and are only hidden by CSS.

Reply:
We don't want all of the code lines' DOM to be loaded first ( rendered ) though, which is the purpose of this PR, hence we also need `v-if` to control the loading.

Reply:
Just condition on `wasCodeLoaded` then. Will there be any cases that a file is active but not loaded?

Reply:
Yup, when the authorship view is initially loaded, `wasCodeLoaded` is false for all files.
Files with < 2000 loc are also not `active` in addition as discussed here https://github.com/reposense/RepoSense/pull/1065#issuecomment-583850317

`wasCodeLoaded` is toggled bit by bit for each file, which loads the page asynchronously.
It's also switched on when the user expands files with >= 2000 loc ( by the expand all button or toggling the file individually )

Reply:
@0blivious I see your point now, thanks!

Just to clarify again, you meant that only the `||` in `file.wasCodeLoaded || file.active` is unecessary right?
ie. It should just be `file.wasCodeLoaded`

Reply:
Yes, you can do sth like v-if="file.wasCodeLoaded"

Reply:
Yes, sth like v-if="file.wasCodeLoaded"